Abracadabra/AbracadabraAPI/AbracadabraAPI/Controllers/AnswersController.cs

```diff
@@ -97,8 +97,8 @@ public async Task<IActionResult> PutAnswer(int id, AnswerViewModel answerViewMod
         }
 
         // POST: api/Answers
-        [HttpPost]
-        [Authorize]
+        [HttpPost]
+        [Authorize(Roles = "User,Admin,Expert")]
```

Comment:
This is okay for the scope of this school project. It filters out the Banned users but in the long run it would be better just to put a check in the methods to see if the user is banned, as we may want to add more roles and we will have to change every method.

Same for all others further down

Reply:
I agree completly, maybe we can build a custom attribute that handles the ban check in the clean up sprint